src/matplotlib-stubs/_typing.pyi

```diff
@@ -15,9 +16,10 @@ PandasScalar = pd.Period | pd.Timestamp | pd.Timedelta | pd.Interval
 Scalar = PythonScalar | PandasScalar
 
 RGBColor = tuple[float, float, float]
-RGBAColor = tuple[float, float, float, float]
+RGBAColor = tuple[np.float_, np.float_, np.float_, np.float_]
+RGBAColorBytes = tuple[np.uint8, np.uint8, np.uint8, np.uint8]
```

Comment:
```suggestion
RGBAColorInt = tuple[np.uint8, np.uint8, np.uint8, np.uint8]
```

I know it comes from `bytes = True` but technically it's a color in the [0, 255]